corehq/apps/style/templates/style/crispy/bootstrap3/field_with_help_bubble.html

```diff
@@ -5,7 +5,7 @@
 {% else %}
     <{% if tag %}{{ tag }}{% else %}div{% endif %}
         id="div_{{ field.auto_id }}" 
-        class="form-group{% if wrapper_class %} {{ wrapper_class }}{% endif %}{% if form_show_errors%}{% if field.errors %} error{% endif %}{% endif %}{% if field.css_classes %} {{ field.css_classes }}{% endif %}"
+        class="form-group{% if wrapper_class %} {{ wrapper_class }}{% endif %}{% if form_show_errors%}{% if field.errors %} has-error{% endif %}{% endif %}{% if field.css_classes %} {{ field.css_classes }}{% endif %}"
```

Comment:
@biyeun @kkrampa is this going to cause any problems if anything was currently relying on the `error` class being set here?